server/routes/varyingLicences/index.ts

```diff
@@ -27,6 +27,13 @@ import PolicyChangesCallbackRoutes from './handlers/policyChangesCallback'
 import PolicyChangeRoutes from './handlers/policyChange'
 import PolicyChangesInputCallbackRoutes from './handlers/policyChangesInputCallback'
 
+function alterResObject() {
```

Comment:
I'm not sure you need this middleware. Could you check the licence.isVariationOf value instead?

Reply:
I don't think it will work with licence.isVariationOf or even licence.isVariation as this won't be present on first run of vary journey. 
    await this.licenceService.updateStatus(licence.variationOf, LicenceStatus.INACTIVE, user) is only executed on POST